### PR TITLE
Save Playwright test results as GitHub artifact

### LINKED ISSUE #50: Need to save E2E test results as an artifact
Path is `test-results` ([source](https://playwright.dev/docs/test-configuration#advanced-configuration))

### PR DESCRIPTION
**Pull request by alexkuc**
Fixes #50

### CODE REVIEW

**Review by tn3rb — APPROVED** [review]
Dude I love reading your code. 
Everything is super easy to follow and understand even though I have zero experience with the libraries being utilized.

I only had one comment pertaining to an error message (ironically about my inability to understand it 😆), but everything else looks fantastic.

File: packages/e2e-tests/src/Artifact.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		const files = await this.getFiles(input, workDir);
+
+		// no need to abort CI runner if artifact upload fails
+		// this method returns true/false which allows the invoking
+		// code to decide if process should be stopped or not
+		const options: UploadOptions = { continueOnError: true, retentionDays: days };
+
+		if (files.length === 0) {
+			core.notice(
+				`Given artifact input '${this.inputToStr(input)}' is empty following working directory '${workDir}'`
```
[the remark this answers is not in the document]
Comment (alexkuc) [reply packages/e2e-tests/src/Artifact.ts:32]: Yeah, it's a bit unclear :/ … I'll update the message!

[the remark this answers is not in the document]
Comment (alexkuc) [reply packages/e2e-tests/src/Artifact.ts]: @tn3rb I've updated the message, please let me know if it is clearer now

**Comment by alexkuc** [comment]
⚠️ force push due to rebase

File: packages/e2e-tests/src/Artifact.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tn3rb) [reply packages/e2e-tests/src/Artifact.ts]: ya that makes sense now, thnx 👍🏻

**Pull request merged**